lib/mix/test/mix/tasks/test_test.exs

```diff
@@ -510,7 +510,9 @@ defmodule Mix.Tasks.TestTest do
         output = mix(["test", "--warnings-as-errors"])
         assert output =~ "2 failures"
         refute output =~ "Test suite aborted after successful execution"
-        output = mix(["test", "--failed"])
+        # missing assertion here
```

Comment:
I could not figure out how to test that the actual Mix Task  is calling ExUnit with the correct options